pkg/pulumiyaml/codegen/gen_program_test.go

```diff
@@ -64,9 +64,9 @@ func (m FakePackage) ResolveResource(typeName string) (pulumiyaml.ResourceTypeTo
 		OtherThing, OtherModule:
 		return pulumiyaml.ResourceTypeToken(typeName), nil
 	default:
-		msg := fmt.Sprintf("Unexpected type token in ResolveResource: %q", typeName)
-		m.t.Logf(msg)
-		return "", fmt.Errorf(msg)
+
+		m.t.Logf("Unexpected type token in ResolveResource: %q", typeName)
+		return "", fmt.Errorf("Unexpected type token in ResolveResource: %q", typeName)
```

Comment:
Let's not introduce duplication if we can help it. 
```suggestion
		msg := fmt.Sprintf("Unexpected type token in ResolveResource: %q", typeName)
		m.t.Logf("%s", msg)
		return "", errors.New(msg)
```

We should do this to all sites where you made this kind of change.